include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
For a specific example, using VS, with the `#ifdef` produces:

    error C3861: 'UNSIGNED_LONGLONGS_EQUAL': identifier not found

And without the `#ifdef` produces:

    error C2664: 'void UtestShell::assertUnsignedLongLongsEqual(cpputest_ulonglong,cpputest_ulonglong,const char *,const char *,int,const TestTerminator &)' : cannot convert argument 1 from 'unsigned __int64' to 'cpputest_ulonglong'
          No constructor could take the source type, or constructor overload resolution was ambiguous